low-level/framework/sspl_reinit

```diff
@@ -59,10 +59,7 @@ chmod 440 /etc/sudoers.d/sspl
 # Set noop = true in puppet conf file to avoid the sspl auth file getting
 # removed by puppet-agent (Ref: LRL-495)
 PUPPET_CONF="/etc/puppetlabs/puppet/puppet.conf"
-[ -f $PUPPET_CONF ] && {
-    PUPPET_NOOP="noop = true"
-    grep -q "$PUPPET_NOOP" $PUPPET_CONF || echo "$PUPPET_NOOP" >> $PUPPET_CONF
-}
+[ -f $PUPPET_CONF ] && conf "ini://$PUPPET_CONF" set "agent>noop=true"
```

Comment:
This script is being called during sspl post_install. At that time, consider only cortx-utils post_install is just completed. 
In this case, does cortx-utils cli become usable once post_install stage completed for cortx-utils? If not, this approach can not work. Please make sure verified this on a clean setup. @sumedhak27 

cortx-utils conf cli will always be

Reply:
Sumedh confirmed this test. Hence approving.